eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -33,7 +33,7 @@ stages:
                       parameters:
                         SourceFolder: ${{parameters.ArtifactName}}
                         TargetFolder: ${{artifact.safeName}}
-                        PackageName: ${{artifact.name}}
+                        PackageName: ${{artifact.name}}-
```

Comment:
I was thinking of adding the dash before `*` in this line
https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/steps/stage-artifacts.yml#L9

Reply:
We need to fix it for docs too - so that's why we are doing it here while passing in the argument rather than on the template

Reply:
if we put the - in the template yaml then it will break in the other steps where we use the template

Reply:
hmm, I don't know these yml files well enough. It just feels weird to have a trailing `-` as part of package names.

Reply:
@KarishmaGhiya I think this fix should go into https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/steps/stage-artifacts.yml#L9. We should handle it in the one place if possible.

Reply:
On that line - for the docs - the packagename is actually the name of the zip file along with the .zip extension. So if i do a - after the package name - `${{parameters.PackageName}}-` it will be something like - `core-http.zip-` which will be incorrect

Reply:
Looks like ```stage-artifacts.yml``` doesn't provide us much value beyond just creating a directory in this case.

Reply:
If we are changing package names here in `archetype-js-release.yml`, would it be better to remove the wild char `*` in `stage-artifacts.yml`, and passing the npm and doc specific names here? like `PackageName: ${{artifact.name}}-*` and `PackageName: $(Documentation.Zip).zip`

Reply:
I agree with @jeremymeng we can definitely fix it like that.

Reply:
yeah we can remove the * in stage-artifacts.yml

Reply:
@scbedd will this be fine for the Docs.MS release steps? What parameter gets passed there?

Reply:
It's this step - https://github.com/Azure/azure-sdk-for-js/pull/9820/files#diff-47954f7c8ecc3cb8e4225ab386a6f44fR92